source/includes/fact-mongodb30-upsert-id.rst

```diff
@@ -1,9 +1,3 @@
 When you execute an :method:`~db.collection.update()` with the ``upsert``
-option, you can no longer use :ref:`document-dot-notation` to select only on
-part of the ``_id`` field. For example, the following will raise an error:
-
-.. code-block:: javascript
-
-   db.collection.update( { "_id.authorID": 1 },
-      { "name": "Robert Frost" },
-      { upsert: true } )
+option, you can not use :ref:`dot notation <document-dot-notation>` to select only
+part of the ``_id`` field.
```

Comment:
Ditto. Given my test examples, the sentence needs to be more precise.  Afterwards, could you have scott code review?


> db.foo.update(       { "_id.authorID": 2 },       { "name": "Robert Frost" },       { upsert: true }    )
WriteResult({
	"nMatched" : 0,
	"nUpserted" : 0,
	"nModified" : 0,
	"writeError" : {
		"code" : 111,
		"errmsg" : "field at '_id' must be exactly specified, field at sub-path '_id.authorID'found"
	}
})
> db.foo.update(       { "_id.authorID": 1 },       { "name": "2Robert Frost" },       { upsert: true }    )
WriteResult({ "nMatched" : 1, "nUpserted" : 0, "nModified" : 1 })